frontend/ui/src/pages/Inbox/MessageInput/InputOptions.tsx

```diff
@@ -64,7 +64,9 @@ export const InputOptions = (props: Props) => {
   useEffect(() => {
     if (!isFileLoaded) {
       setInputFile(null);
-      inputRef.current.value = null;
+      if (inputRef && inputRef.current) {
+        inputRef.current.value = null;
+      }
```

Comment:
this was creating an error: when the component mounts, ref.current is not yet initialised